src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: APM data in infrastructure monitoring
+title: Infrastructure monitoring within APM 
```

Comment:
I think 'within' is a bit awkward, might just be me. Maybe 'Infrastructure data on APM Summary page'? 
Also ensure you align the nav doc title with this title.